package tasks2;

import java.sql.SQLException;

public class ConnectionClass {
	
	public void openConnection() {
		
		try {
			
			
		}catch(SQLException e) {
			System.out.println("connection non aboutie à cause de : " + e.getMessage());
		}catch(ClassNotFoundException e) {
			System.out.println("connection non aboutie à cause de : " + e.getMessage());
		}
	}
	
	public void closeConnection() {
		
		try {
			connection.close();
		}catch(SQLException e) {
			System.out.println("connection non aboutie à cause de : " + e.getMessage());
		}
	}

}
